inc/notificationtemplate.class.php

```diff
@@ -250,7 +250,11 @@ function getTemplateByLanguage(NotificationTarget $target, $user_infos=array(),
          //Restore default language
          $_SESSION["glpilanguage"] = $bak_language ;
          Session::loadLanguage();
-         $_SESSION['glpi_dropdowntranslations'] = $bak_dropdowntranslations ;
+         if ($bak_dropdowntranslations !== null) {
```

Comment:
the if/else block could be not needed.
In fact, in the current diff, we can see at l231, the $_SESSION['glpi_dropdowntranslations'] assignement.

So, in cron mode, $_SESSION['glpi_dropdowntranslations'] with a null/language value may be not important.

Otherwise, we need to condition the l231 also